README.md

```diff
@@ -1,14 +1,36 @@
 # py-dmidecode
 
-Small tool that parse the ouput of the dmidecode command
+Small tool that parses ouput of dmidecode command
 
 ## How to use
 ```python
-import dmidecode
+from dmidecode import DMIDecode
+import subprocess
 
-# parse the full output
-data = dmidecode.parse()
+# run dmidecode and store its output
```

Comment:
I would move these 2 line to the line 29 in the second example where you show how to use the DMIParse object.

Reply:
Oh, I forgot to remove that part. Running dmidecode is already covered in DMIDecode's __init__(). Removing...